app/views_redesigned/async_property_card_footers/show.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="row align-items-center text-center pt-4">
+  <% if @property.restrict_to_sponsor_campaigns? %>
+
+    <div class="col-4 u-ver-divider">
+      <h5 class="small font-weight-normal mb-0">
+        <span class="d-block">Total</span>
+        <span class="d-block">Clicks</span>
+      </h5>
+      <div class="text-dark mt-2"><%= number_with_delimiter @property.clicks_count(@start_date, @end_date) %></div>
+    </div>
+    <div class="col-4">
+      <h5 class="small font-weight-normal mb-0">
+        <span class="d-block">Current</span>
+        <span class="d-block">Advertiser</span>
+      </h5>
+      <div class="text-dark mt-2"><%= @property.current_sponsor_campaign&.user&.name || "none" %></div>
+    </div>
+    <div class="col-4">
+      <h5 class="small font-weight-normal mb-0">
+        <span class="d-block">Current</span>
+        <span class="d-block">Campaign</span>
+      </h5>
+      <div class="text-dark mt-2"><%= @property.current_sponsor_campaign&.name || "none" %></div>
+    </div>
+
+  <% else %>
+
+    <div class="col-4 u-ver-divider">
```

Comment:
`u-ver-divider` is not a class in our new theme. You could copy it in as a utility class or perhaps there is another way we could achieve this?

Reply:
I actually looked at this. Its a fairly small stand alone Sass file. Maybe 30-50 lines. It looks decently clean without porting this and with Eric going through things after us, I wasn't sure how much we want to dive into UI things. Let me know your thoughts @andrewmcodes @hopsoft @coderberry

Reply:
I’d say first make sure our new theme doesn’t have something similar. Then if you think it is something that adds value pull the stylesheet in.

Reply:
Diving into this it will take some time to port. The `u-ver-divider` style is only 50 lines but relies on other scss partials. Parent and sibling divider styles, media-breakpoints, variables, etc.

We may want to backlog this for another PR when we cleanup any vendor or dangling stylesheets.

Reply:
Diving into this it will take some time to port. The `u-ver-divider` style is only 50 lines but relies on other scss partials. Parent and sibling divider styles, media-breakpoints, variables, etc.

We may want to backlog this for another PR when we cleanup any vendor or dangling stylesheets.

Reply:
Just remove these classes and we’ll move forward from there.